components/AliadosCarousel.js

```diff
@@ -0,0 +1,38 @@
+import { Swiper, SwiperSlide } from 'swiper/react';
+import 'swiper/swiper.min.css';
+import SwiperCore, { Pagination } from 'swiper/core';
+import AliadosSlide from './AliadosSlide';
+
+SwiperCore.use([Pagination]);
+
+export default function AliadosCarousel({ data }) {
+  const { carouselComponents } = data[0].fields;
+  const { brandLogos } = data[0].fields;
+
+  return (
+    <>
+      <Swiper
+        speed={600}
+        pagination={{
+          clickable: true,
+          type: 'bullets',
+          bulletElement: 'span',
+          bulletClass: 'aliadosBullet',
+          bulletActiveClass: 'aliadosBullet-active',
+        }}
+        className="aliadosCarousel"
+      >
+        {carouselComponents.map((slide) => (
+          <SwiperSlide>
```

Comment:
This map without a key are going to show this warning
`Each child in a list should have a unique "key" prop`

You could look for an identifier and put something like
`<SwiperSlide key="identifier" ...`